### PR TITLE
I had lunch today and thoucht the docs wer bad

### PR DESCRIPTION
**Pull request by Aksel-Rasmussen**
We love github

### CODE REVIEW

**Comment by Aksel-Rasmussen** [comment]
I don''t like this change

File: README.md [hunk 1]
```diff
@@ -37,3 +37,6 @@ In addition to this GitHub site, we will be using a few other resources this yea
 I am a little disappointed that the meeting I made a full soundtrack for was snubbed when Oscar nominations rolled around. The outro sequence was possibly the best thing in the history of things. 
 
 In a fit of protest, I've used the silent film genre for a more recent bonus video, but without the typical accompanying soundtrack. Take that, AMPAS.
+
+
+We love github
```
Comment (sql-sith) [inline README.md:42]: I'm happy for you.

Comment (Aksel-Rasmussen) [reply README.md:42]: ok, but it's not just me

**Review by sql-sith — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (Aksel-Rasmussen) [reply README.md:42]: i'm fine with how this strund out

**Pull request merged**